### PR TITLE
feat:added cards

### PR DESCRIPTION
**Pull request by Aazam10**
Hi I have added cards can you please review

### CODE REVIEW

**Review by RahullRawat — APPROVED** [review]
Everything looks good aazam

**Review by Kevin-Solomon — APPROVED** [review]

File: Card/card.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+}
+.card-button {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+.card-icons {
```
Comment (VishalPatil18) [inline Card/card.css:45]: **FEAT:** Here you can give these icons a "cursor: pointer;" as doing so will help the user know that it's a clickable item. With the default cursor, one cannot understand whether it's a button or just an image.

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  right: 0;
+  z-index: 2;
+  color: white;
+  cursor: pointer;
+  height: 1.5rem;
+  width: 1.5rem;
```
Comment (VishalPatil18) [inline Card/card.css:86]: **Suggestion:** A small suggestion here is that as this class has many properties, you can sort them in ascending order alphabetically as Vikram sir had suggested. It helps in debugging or understanding the code easily.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
Apart from one fix and one suggestion, the code works fine. Great work Aazam 👍

**Review by kart-c — APPROVED** [review]

File: Card/card.css (hunk 2 above)
Comment (Aazam10) [reply Card/card.css:86]: Hi Vishal I have added the cursor property and tried to refactor code in alphabetical order for some classes will take care of that in future .Can you please take a look and review the PR. Thanks!

**Review by VishalPatil18 — APPROVED** [review]
Perfect. The code works well. Great job Aazam 👍

**Pull request merged**